lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -60,8 +58,6 @@ func NewImageCache() *ImageCache {
 
 // Update adds new images to the cache
 func (c *ImageCache) Update(client *client.PortLayer) error {
-	c.m.Lock()
-	defer c.m.Unlock()
 
 	CacheNotUpdated = true
```

Comment:
I have confuse. I thought in our slack discussion we decided we needed to keep this?